archive_upload/handlers/dsmc_handlers.py

```diff
@@ -653,55 +649,51 @@ def post(self, runfolder):
         path_to_archive_root = self.config["path_to_archive_root"]
         path_to_archive = os.path.abspath(
             os.path.join(path_to_archive_root, runfolder) + "_archive")
-        exclude_dirs = []
-        exclude_extensions = []
 
         # Messages
         invalid_body_msg = "Invalid body format."
         missing_rm_msg = "The `remove` field must be a boolean."
-        exclude_dirs_msg = "The `exclude_dirs` field must be a comma-separated string."
-        exclude_extensions_msg = "The `exclude_extensions` field must be a comma-separated string."
 
         try:
             request_data = json.loads(self.request.body)
         except (ValueError, KeyError):
             raise ArchiveException(reason=invalid_body_msg, status_code=400)
 
-        remove = False
-        if "remove" in request_data:
-            remove = request_data["remove"]
+        remove = request_data.get("remove", False)
+        if remove:
             # Booleans may arrive in either in:
             # 1) JSON format (true), or
             # 2) stringified Python format ("True") (e.g. from the Irma archiving workflow)
             # TODO: Remove support for format 2, see DEVELOP-1024 // ML, 2021-01
             try:
                 if not isinstance(remove, bool):
                     remove = eval(request_data["remove"])
```

Comment:
We need to remove this `eval`. I think `bool` should work the same but is safe.

I also think we should add a second reviewer here since this includes remove logic :sweat_smile:

Reply:
Adding @matrulda